addons/safeStart/functions/fnc_toggleSafeStart.sqf

```diff
@@ -26,4 +26,5 @@ if (_makeSafe) then {
 } else {
     missionNamespace setVariable [QGVAR(startTime_PV), -1, true];
     ["potato_safeStartOff"] call CBA_fnc_globalEvent;
+    if (hasInterface) then { ["potato_adminMenu_resetMarkers", [player]] call cba_fnc_localEvent; }; // Resets BFT for missions using staged load-in
```

Comment:
this func only runs on server so it's fail the hasinterface
and potato_adminMenu_resetMarkers seems to want `player`, but I'm not sure why and that could probably be dropped

Reply:
is there a place you recommend to run this? I was going for the "dirty" method here and just making it reset markers when SafeStart is toggled, but really it would be more ideal if people's markers got a reset when they spawned (so they could see where PLT/COY is during SafeStart for briefing etc...). So maybe in a `PostInit` somewhere when players spawn? or even building something into the `markers` submod itself - let me know what your recommendation would be and I can build it from there

Reply:
disregard, see conversation in #scripting on discord, best catch-all is just to run it when safestart toggles